src/styles/index.scss

```diff
@@ -1,3 +1,139 @@
+$main-text-color: #060b35;
+$secondary-text-color: #616070;
+$button-color: #00acdc;
+$button-text-color: #fff;
+
 body {
+  font-family: Roboto, sans-serif;
+  padding: 0;
   margin: 0;
 }
+
+.main {
+  display: flex;
+  justify-content: center;
+  align-items: start;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: 198px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 0 0;
+  }
+
+  &__tittle {
+    margin: 40px 16px 0;
+
+    color: $main-text-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    margin: 4px 16px 0;
+
+    color: $secondary-text-color;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 16px;
+    width: 166px;
+    height: 40px;
```

Comment:
it's better not hardcode height for elements with content inside. Just use paddings